test/setup/make_source.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id=$(terraform output project_id)
+sa_json=$(terraform output sa_key)
+organization_id=$(terraform output organization_id)
+folder_exclude=$(terraform output folder_exclude)
+folder_id=$(terraform output folder_id)
+folder_1_id=$(terraform output folder_1_id)
+folder_2_id=$(terraform output folder_2_id)
+project_exclude=$(terraform output project_exclude)
+
+# shellcheck disable=SC2086,SC2154
+{ echo "export TF_VAR_project_id='$project_id'"; \
+echo "export TF_VAR_ORGANIZATION_ID='$organization_id'"; \
+echo "export TF_VAR_FOLDER_EXCLUDE='$folder_exclude'"; \
+echo "export TF_VAR_FOLDER_ID='$folder_id'"; \
+echo "export FOLDER_ID='$folder_1_id'"; \
+echo "export FOLDER_2_ID='$folder_2_id'"; \
+echo "export PROJECT_ID='$project_id'"; \
+echo "export PROJECT_EXCLUDE='$project_exclude'"; \
+echo "export PROJECT_CONSTRAINT_DENY_ALL='constraints/compute.trustedImageProjects'"; \
```

Comment:
We shouldn't have any static values in this file. make_source is only necessary for generating values that change.

Please move all the static value back to the old location.

Reply:
@morgante. I agree, the constraint variables have been _mistakenly_ moved to dynamic ones.

@kopachevsky @nick4fake,
Since current Cloud Build integration with GitHub doesn't support logs, further development of Cloud Build integration with this module would require approved access to the mentioned infrastructure component on the client side.

@kopachevsky, your Team take over of the task or GL guidance on further actions on it is required. As always, I'll be more than happy to help with any challenges GL may face ;)